azkaban-common/src/main/java/azkaban/sla/SlaOption.java

```diff
@@ -302,23 +313,26 @@ public String createSlaMessage(final ExecutableFlow flow) {
    * @return the job level SLA options.
    */
   static public List<SlaOption> getJobLevelSLAOptions(List<SlaOption> options) {
-    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.JOB).collect
-        (Collectors.toList());
+    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.JOB)
+        .collect
```

Comment:
Could 317 and 218 be condensed to 1 line.

Reply:
sure